summary_workflows/quantification/quantification_dockers/q2_metrics/compute_metrics.py

```diff
@@ -54,33 +53,30 @@ def compute_metrics(input_participant, gold_standards_dir, challenge_types, part
         # get metrics dataset
 #        metrics_data = pandas.read_csv(os.path.join(gold_standards_dir, challenge + ".txt"),
 #                                       comment="#", header=0, sep='\t',index_col=0)
-        metrics_data=os.path.join(gold_standards_dir, "siControl_R2.MACEseq.mm10.bed") ##PLEASE CHANGE THIS LATER
+        metrics_data=os.path.join(gold_standards_dir, challenge + ".bed") ##NO LONGER HARD-CODED
 
         # metric on the number of matched sites
-        window = 15
+        #window = 15
         match_with_gt_run = Leo_MatchPAsites_GT.match_wtih_gt(input_participant,metrics_data,window)
         merged_bed_df, n_matched_sites, n_unmatched_sites = match_with_gt_run[0], match_with_gt_run[1], match_with_gt_run[2]
+        percent_matched=n_matched_sites/(n_matched_sites+n_unmatched_sites)
```

Comment:
This metric is never mentioned in the Q2 specification (there, only correlation is mentioned, also in the specification that has been updated in the meanwhile by @daneckaw (https://github.com/iRNA-COSI/APAeval/pull/210/files#diff-752c23db2dc0d0c543326d5e5ca8bb7d288d17f8b2c70a7388b8332c585d6d7f)).    
Either this has to be added to the specification, or removed here. I know it appears a couple of times downstream, as the OEB .jsons for the plots also specify this "percent_matched" as a metric.

Reply:
You are right, we said we would only output the correlation, we need to fix this.